Week3/mybusiness.sql

```diff
@@ -0,0 +1,189 @@
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `departments`
+--
+
+DROP TABLE IF EXISTS `departments`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `departments` (
+  `dep_number` int(11) NOT NULL AUTO_INCREMENT,
+  `employee_id` int(11) NOT NULL,
+  `department_name` varchar(50) NOT NULL,
+  PRIMARY KEY (`dep_number`),
+  KEY `employee_id` (`employee_id`),
+  CONSTRAINT `departments_ibfk_1` FOREIGN KEY (`employee_id`) REFERENCES `employees` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `departments`
+--
+
+LOCK TABLES `departments` WRITE;
+/*!40000 ALTER TABLE `departments` DISABLE KEYS */;
+INSERT INTO `departments` VALUES (1,1,'IT');
+/*!40000 ALTER TABLE `departments` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `employees`
+--
+
+DROP TABLE IF EXISTS `employees`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `employees` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `full_name` varchar(50) NOT NULL,
+  `date_of_birth` date NOT NULL,
+  `address` varchar(50) NOT NULL,
+  `phone_number` varchar(50) NOT NULL,
+  `e_mail` varchar(50) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `employees`
+--
+
+LOCK TABLES `employees` WRITE;
+/*!40000 ALTER TABLE `employees` DISABLE KEYS */;
+INSERT INTO `employees` VALUES (1,'emre','1991-06-15','Netherland','12344566','earas446@gmail.com');
+/*!40000 ALTER TABLE `employees` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `project_info`
+--
+
+DROP TABLE IF EXISTS `project_info`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `project_info` (
+  `project_id` int(11) NOT NULL,
+  `department_id` int(11) NOT NULL,
+  KEY `project_id` (`project_id`),
+  KEY `department_id` (`department_id`),
+  CONSTRAINT `project_info_ibfk_1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`project_id`),
+  CONSTRAINT `project_info_ibfk_2` FOREIGN KEY (`department_id`) REFERENCES `departments` (`dep_number`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `project_info`
+--
+
+LOCK TABLES `project_info` WRITE;
+/*!40000 ALTER TABLE `project_info` DISABLE KEYS */;
+/*!40000 ALTER TABLE `project_info` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `project_team`
+--
+
+DROP TABLE IF EXISTS `project_team`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `project_team` (
```

Comment:
Again, a many-to-many of projects to employees. But that's not really a team, per se - you'll end up with one record per employee per project, and the table name implies that one record would be one team, which is not the case.